tests/Ember/sfc.gts

```diff
@@ -0,0 +1,36 @@
+// I am top level comment in this file.
+import z from 'z';
+import threeLevelRelativePath from '../../../threeLevelRelativePath';
+import sameLevelRelativePath from './sameLevelRelativePath';
+import thirdParty from 'third-party';
+import oneLevelRelativePath from '../oneLevelRelativePath';
+import otherthing from '@core/otherthing';
+import abc from '@core/abc';
+import twoLevelRelativePath from '../../twoLevelRelativePath';
+import component from '@ui/hello';
+import fourLevelRelativePath from '../../../../fourLevelRelativePath';
+import * as a from 'a';
+import type RouterService from '@ember/routing/router-service';
```

Comment:
let's add a few more styles like `import type {T} from...` and `import a, type {T}`, etc.  Just for good measure.